tests/test_scrape.py

```diff
@@ -71,5 +72,5 @@ def test_data():
         json=sample_data
     )
 
-    prom_data = data(enrollment, token, month='2017-03')
-    assert prom_data.count('azure_costs') == 4
+    prom_data = data(enrollment, token, metric_name, month='2017-03')
+    assert prom_data.count(metric_name) == 4
```

Comment:
Well, the sample data contains just 2 records. The metric name occurs another 2 times in the description, so I am just counting occurrence of the work in the Prometheus compatible output.

It is simpler than parsing the counters and adding them up. Do you think it would provide more validation if I'd change that?